repos/omeka-i18n-module/nav.json

```diff
@@ -0,0 +1 @@
+[{"type":"page","data":{"id":10,"label":""},"links":[]},{"type":"browse","data":{"label":"Browse","query":""},"links":[]}]
```

Comment:
@garyttierney Any idea what this export is?

Reply:
Ah, this was me debugging what the nav structure looked like when implementing `TranslatableNavigationResource`. Will remove.